Dockerfile

```diff
@@ -57,6 +57,7 @@ ARG BUILD_DEPENDENCIES="              \
         freerdp2-dev                  \
         gcc                           \
         libcairo2-dev                 \
+        libgcrypt-dev                 \
```

Comment:
Is this the correct package name? I don't see it in the Debian package search.

Reply:
This is what I was going off of:
https://packages.debian.org/buster/libgcrypt-dev

Looks like it is a "virtual package" for libgcrypt20-dev?